include/tc/aten/utils-inl.h

```diff
@@ -21,6 +21,7 @@
 #include <ATen/ATen.h>
 #include <ATen/DLConvertor.h>
 namespace tc {
+namespace {
```

Comment:
I believe these are handy functions that are needed somewhere else like autotuner. perhaps we don't need to put it in anonymous namespace.

Reply:
It's to avoid the symbol being created multiple times which reults in linking errors.
The alternative is to go back to have a tc_aten library which for the purpose of these 3 functions does not seem necessary.